cogs5e/gametrack.py

```diff
@@ -471,10 +486,9 @@ async def customcounter(self, ctx, name=None, *, modifier=None):
         await character.commit(ctx)
 
         delta = f"({counter.value - old_value:+})"
+        out = f"{str(counter)} {delta}\nRoll: {roll_result}"
```

Comment:
Will this display the roll even if the modifier is not a roll (e.g. `!cc blah -1`)?

Reply:
Currently yes, I considered making it skip the roll portion if it was just an int instead. Would that be preferable?